File: examples/primes.java
public class Main {
    static boolean IsPrime(int n)
    {
        for ( int i = 2; i <= n / 2; ++i) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String args[]) {   
        for (int i = 2; i< 100; i++)
        {
            if (IsPrime(i))
               System.out.println(i);
        }
    }
}

File: examples/primes.go
package main

import (
	"fmt"
)
func IsPrime(n int) bool {
    for i:=2; i<=n/2; i++ {
        if (n % i == 0) {
           return false;
        }
    }
 return true;
}

func main() {
    for i:= 2; i<100; i++ {
        if (IsPrime(i)) {
           fmt.Println(i);
		}
    }
}